rdmo/questions/migrations/0069_catalogs_add_editors_alter_sites.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.16 on 2023-01-24 13:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sites', '0002_alter_domain_unique'),
+        ('questions', '0068_meta'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='catalog',
+            name='editors',
+            field=models.ManyToManyField(blank=True, help_text='The sites that can edit this catalog (in a multi site setup).', related_name='questions_catalog_editors', related_query_name='questions_catalog_editors', to='sites.Site', verbose_name='Edited by Sites'),
+        ),
+        migrations.AlterField(
```

Comment:
Do you know what changed here?

Reply:
I renewed the `migrations` and it adds the `editors` field only